README.md

```diff
@@ -465,7 +468,7 @@ JSON inventory method
  
 - **Ansible Playbook:** 
     - **playbook_json.ces.yml**
-         -  Spectrum Scale Core, ZIMON, GUI, protocol (NFS, SMB), CallHome and Scale file audit logging. 
+         -  Roles Include: Core, zimon, GUI, Protocol (NFS, SMB), callhome and scale_fileauditlogging. 
```

Comment:
Instead of `scale_fileauditlogging` I'd suggest to call the role `fileauditlogging` (without 'scale_') for consistency.